site/src/App/SubNavigation/SubNavigation.tsx

```diff
@@ -125,13 +37,21 @@ export const SubNavigation = ({ onSelect }: SubNavigationProps) => {
     }
   };
 
+  const { pathname: currentPath } = useLocation();
+  const isActive = useMemo(
+    () => (path: string) =>
+      Boolean(matchPath({ path, end: true }, currentPath)),
+    [currentPath],
+  );
+
   return (
     <Stack space="large">
-      <SubNavigationGroup
+      <SideNavigationGroup
         items={[
           {
             name: 'Releases',
             path: '/releases',
+            active: Boolean(useMatch({ path: '/releases', end: true })),
```

Comment:
```suggestion
            active: isActive('/releases'),
```
(adding this suggestion will require removing `useMatch` import as it will be unused)